### PR TITLE
Add correct enum names for ProductionID, ObjectID, DrawableID, WaypointID and FormationID based on debug strings in BFME2. This is a simple renaming of an enum so should require minimal testing.

### PR DESCRIPTION
**Pull request by jonwil**

### CODE REVIEW

File: src/game/client/draw/drawable.cpp [hunk 1]
```diff
@@ -1577,7 +1577,7 @@ void Drawable::Xfer_Snapshot(Xfer *xfer)
     if (m_object != nullptr) {
         object_id = m_object->Get_ID();
     } else {
-        object_id = OBJECT_UNK;
+        object_id = INVALID_ID;
```
Comment (xezon) [inline src/game/client/draw/drawable.cpp]: INVALID_ID reads ambiguous. Perhaps call this INVALID_OBJECT_ID?

Comment (tomsons26) [reply src/game/client/draw/drawable.cpp]: well indeed its INVALID_ID
BFME1 also shows it

Comment (tomsons26) [reply src/game/client/draw/drawable.cpp]: while probably should go with INVALID_OBJECT_ID for consistency with others

Comment (jonwil) [reply src/game/client/draw/drawable.cpp]: Fixed.

**Review by xezon — APPROVED** [review]

**Pull request merged**